runtime/themes/ao.toml

```diff
@@ -0,0 +1,212 @@
+# Theme: Ao
+# Author: YardQuit
+
+# SYNTAX HIGHLIGHTING
+"attribute" = { fg = "yellow" }                                                                      #
+"type" = { fg = "white" }                                                                            # types e.g. return type: Result, String, etc
+"type.builtin" = { fg = "white" }                                                                    # primitive types provided by the language (int, usize)
+"type.parameter" = { fg = "white" }                                                                  # generic type parameter (T)
+"type.enum" = { fg = "white" }                                                                       #
+"type.enum.variant" = { fg = "white" }                                                               #
+"constructor" = { fg = "orange" }                                                                    # e.g. Ok, Err, etc
+"constant" = { fg = "blue" }                                                                         # const name 
+"constant.builtin" = { fg = "blue" }                                                                 #
```

Comment:
Quite a lot of these seem redundant. More specific scopes will always fall back on the more general scopes so if constant is already blue there's no need to declare constant.builtin that's also blue

Reply:
Thanks for your comment. I have updated it by commenting out what I then believed to be redundant settings. If confirmed, I can remove all commented lines to clean it up.

Reply:
Looks correct! Similar thing applies for the other theme, I think your other PR https://github.com/helix-editor/helix/pull/10381 leaked into this one

Reply:
Thanks, I have removed redundancy from both themes